client/src/store/features/analysis/index.ts

```diff
@@ -24,6 +24,7 @@ export type AnalysisState = {
     materials: Option[];
     origins: Option[];
     suppliers: Option[];
+    interventionType: number;
```

Comment:
we only have three interventions type, so in case you want to use numbers, it should be limited. Anyway, I would prefer to be more explicit and use strings (slugs).